django_elasticsearch_dsl/search.py

```diff
@@ -13,12 +13,17 @@ def _clone(self):
         s._model = self._model
         return s
 
-    def to_queryset(self, keep_order=True):
+    def filter_queryset(self, queryset, keep_search_order=True):
```

Comment:
Thanks for the update. As we already have `get_queryset` method, you can make the filter there. So we do not need another method for `filter_queryset`